jupyter_rsession_proxy/__init__.py

```diff
@@ -29,13 +32,46 @@ def setup_rserver():
     def _get_env(port):
         return dict(USER=getpass.getuser())
 
+    def db_config():
+        '''Create a temporary directory to hold rserver's database.'''
+        db_dir = tempfile.TemporaryDirectory()
+        # create the rserver database config
+        db_conf = dedent("""
+            provider=sqlite
+            directory={directory}
+        """).format(directory=db_dir.name)
+        f = tempfile.NamedTemporaryFile(mode='w', delete=False)
+        db_config_name = f.name
+        f.write(db_conf)
+        f.close()
+        return db_config_name
+
     def _get_cmd(port):
-        return [
+        cmd = [
             get_rstudio_executable('rserver'),
+            '--auth-none=1',
+            '--www-frame-origin=same',
             '--www-port=' + str(port),
-            '--www-frame-origin=same'
```

Comment:
Is this still needed for older versions of rstudio?

Reply:
I'm not removing it, I just alphabetized the options to make it easier to find them. :)

Reply:
aaaaaah ok :)